flink/v1.17/build.gradle

```diff
@@ -152,6 +152,9 @@ project(":iceberg-flink:iceberg-flink-runtime-${flinkMajorVersion}") {
     }
     implementation(project(':iceberg-nessie')) {
       exclude group: 'com.google.code.findbugs', module: 'jsr305'
+      // Nessie brings Jackson dependencies of version 2.15.0 or later.
+      // These are multi-release jars and doesn't work with Java-8 shadowJar plugin.
```

Comment:
I'm actually not sure I understand what this comment means. Could you elaborate please?

Reply:
Oh, if we just bump the Nessie version, the build will fail for shadowJar task for runtime hive, runtime flink, runtime spark with the error mentioned [here](https://github.com/apache/iceberg/actions/runs/5013686487/jobs/8987072724)

The reason for failure is Nessie brings Jackson 2.15.1 as a dependency for runtime shadow jar tasks. But Jackson 2.15.0 onwards releases [multi-release jars](https://github.com/FasterXML/jackson-core/issues/897) (packs multiple versions of java classes).   Hence, the shadowJar tasks running on java 8 fails to build the uber jar.

So, I am excluding Nessie's Jackson version. So, it uses Iceberg's jackson version  2.14.1 for building the uber jar.